ppr-ui/src/components/mhrRegistration/ReviewConfirm/HomeLocationReview.vue

```diff
@@ -62,14 +62,98 @@
             </v-row>
           </template>
 
-          <!-- Other PID Type -->
+          <!-- Reserve -->
+          <template v-if="getMhrRegistrationLocation.otherType === HomeLocationTypes.OTHER_RESERVE">
+            <v-row no-gutters class="px-6 pt-1">
+              <v-col cols="3" class="pt-1">
+                <h3>Legal Land Description</h3>
+              </v-col>
+              <v-col v-if="getMhrRegistrationLocation.bandName || getMhrRegistrationLocation.reserveNumber"
+                cols="9" class="pt-1"
+                >
```

Comment:
Minor formatting here, if it can't all fit on one line, new lines for each condition usually. Also, the `>` chevron to align with the tag.

An example but if you wanted to keep some props inline with the tag, i'm fine with that too, but it might throw the props out of alignment a bit.
<img width="514" alt="Screen Shot 2022-12-08 at 1 00 18 PM" src="https://user-images.githubusercontent.com/53542131/206566485-97695984-2241-4a0f-8de5-b73dbfe9b4d5.png">